### PR TITLE
CI workflows: delete legacy tasks using the old binary

### PR DESCRIPTION
**Pull request by jmontesi**
The same tasks using the new "certsuite" binary are already working as expected, so there is no need to duplicate the verification.

Also, in line with the recent renaming some "tnf" references are changed to "certsuite".

### CODE REVIEW

File: .github/workflows/pre-main.yaml [hunk 1]
```diff
@@ -13,26 +13,17 @@ permissions:
...
-  ON_DEMAND_DEBUG_PODS: false
+  CERTSUITE_CONFIG_DIR: /tmp/certsuite/config
+  CERTSUITE_OUTPUT_DIR: /tmp/certsuite/output
+  SMOKE_TESTS_LOG_LEVEL: debug
+  SMOKE_TESTS_LABELS_FILTER: all
   TERM: xterm-color
   CM_BIN: /usr/local/bin/checkmake
   CM_URL_LINUX: https://github.com/mrtazz/checkmake/releases/download/0.2.2/checkmake-0.2.2.linux.amd64 # yamllint disable-line
-  SMOKE_TESTS_LABELS_FILTER: all
-  SKIP_PRELOAD_IMAGES: true
```
Comment (sebrandon1) [inline .github/workflows/pre-main.yaml:35]: I think we need to keep this flag `SKIP_PRELOAD_IMAGES`.

**Review by sebrandon1 — COMMENTED** [review]
One comment

File: .github/workflows/pre-main.yaml (hunk 1 above)
Comment (jmontesi) [reply .github/workflows/pre-main.yaml:35]: Done.

**Review by sebrandon1 — APPROVED** [review]

File: .github/workflows/pre-main.yaml [hunk 2]
```diff
@@ -226,47 +218,35 @@ jobs:
 
       # Perform smoke tests.
       - name: 'Test: Run test suites'
-        run: TNF_LOG_LEVEL=${TNF_SMOKE_TESTS_LOG_LEVEL} ./run-cnf-suites.sh -l "${SMOKE_TESTS_LABELS_FILTER}"
+        run: ./certsuite run --label-filter="${SMOKE_TESTS_LABELS_FILTER}" --output-dir="certsuite-out" --log-level=${SMOKE_TESTS_LOG_LEVEL}
```
Comment (rdavid) [inline .github/workflows/pre-main.yaml:221]: No need quotes for `certsuite-out` and need for `${SMOKE_TESTS_LOG_LEVEL}`.

File: .github/workflows/pre-main.yaml [hunk 3]
```diff
@@ -226,47 +218,35 @@ jobs:
...
-        run: make build-certsuite-tool
+        run: rm -f certsuite-out/*.tar.gz
 
       - name: Check the smoke test results against the expected results template
-        run: make check-results
+        run: ./certsuite check results --log-file="certsuite-out/certsuite.log"
 
       - name: 'Test: Run preflight specific test suite'
-        run: TNF_LOG_LEVEL=${TNF_SMOKE_TESTS_LOG_LEVEL} ./run-cnf-suites.sh -l "preflight"
+        run: ./certsuite run --label-filter="preflight" --log-level=${SMOKE_TESTS_LOG_LEVEL}
```
Comment (rdavid) [inline .github/workflows/pre-main.yaml:238]: same comment on quates.

**Review by rdavid — APPROVED** [review]

File: .github/workflows/pre-main.yaml (hunk 2 above)
Comment (jmontesi) [reply .github/workflows/pre-main.yaml]: Done.

File: .github/workflows/pre-main.yaml (hunk 3 above)
Comment (jmontesi) [reply .github/workflows/pre-main.yaml]: Done.

**Pull request merged**